keyboards/pearl/keymaps/cijanzen/config.h

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2017 Luiz Ribeiro <luizribeiro@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef PEARL_CONFIG_H
```

Comment:
You shouldn't have the entire file here.  This really should be just: 
```c
#ifndef CONFIG_USER_H
#define CONFIG_USER_H

#include QMK_KEYBOARD_CONFIG_H
#define NO_PRINT

#define BACKLIGHT_LEVELS 3

#define RGBLIGHT_SLEEP

#endif
```

Reply:
ah yes, that makes sense. I'll go ahead and make those changes. Wasn't aware that it would look at both keymap and keyboard files and then use keymap overrides for precedence.

Reply:
Yup.  And the way it was setup, it *may* have been ignoring your file (the `ifndef` stuff).